mailing/sendgrid.go

```diff
@@ -40,32 +50,30 @@ func (s *sendgridEmailService) Send(ctx context.Context, sender string, recipien
 	return nil
 }
 
-// prepareSendgridMailV3 prepares the input values used for sending an email.
-func prepareSendgridMailV3(sender string, recipients []string, cc []string, bcc []string, subject string, htmlContent string) *mail.SGMailV3 {
-	p := mail.NewPersonalization()
-	p.AddFrom(mail.NewEmail("", sender))
-	p.AddTos(parseSendgridEmails(recipients)...)
-	p.AddCCs(parseSendgridEmails(cc)...)
-	p.AddBCCs(parseSendgridEmails(bcc)...)
-	p.Subject = subject
+// NewSendgridEmailSender initializes a new Sender with a sendgrid client. It will send emails using Go templates.
+// See NewTemplateSender for conveniently handling Go templates in your project.
+func NewSendgridEmailSender(client sendgridSender) Sender {
+	return newSendgridEmailSenderWithContentInjector(client, injectHTMLContent)
```

Comment:
Since `newSendgridEmailSenderWithContentInjector` is a private function and there is no other private constructor, we can drop the `WithContentInjector` suffix to make this a little more succint.

```suggestion
	return newSendgridEmailSender(client, injectHTMLContent)
```